cypress/page/login.page.ts

```diff
@@ -0,0 +1,26 @@
+class LoginPage {
+    private emailField: string;
+
+    private passwordField: string;
+
+    private submitBtn: string;
+
+    constructor() {
+      this.emailField = "#email";
+      this.passwordField = "#passwd";
+      this.submitBtn = "[id=SubmitLogin]";
+    }
+
+    public fillEmailField(): void {
+      cy.get(this.emailField).type("aperdomobo@gmail.com");
+    }
+
+    public fillPasswordField(): void {
+      cy.get(this.passwordField).type("WorkshopProtractor");
+    }
+
+    public submitBtnEvent(): void {
+      cy.get(this.submitBtn).click();
+    }
```

Comment:
these three methods can be just one, and that method can receive the password and email as parameters